definitions/divorce/json/AuthorisationCaseState.json

```diff
@@ -1336,6 +1336,13 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "DNDrafted",
+    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
This should not go to prod

Reply:
Could we maybe hide it behind the superuser role in prod?